### DISCUSSION TITLE
Popup input error

### DISCUSSION THREAD

**Discussion by mule-stand** [ariakit/ariakit#3453] [Questions] [answered]
I would like to know what would be the best choice from an accessibility point of view for displaying an error during user input. The idea is that when typing into an input field I want to show some sort of error if the input is wrong, but I want to show it as a popup error, not somewhere below the input field. I was thinking of using Tooltip for this, but I'm not sure if it's a good choice in terms of accessibility. Maybe you can give me a hint. It would be good to see an example, including for example with FormInput and FormError or just some input field outside the form. Thanks in advance!

**Comment by mule-stand** [comment]
pls

**Comment by diegohaz** [comment, marked as answer]
> I was thinking of using Tooltip for this, but I'm not sure if it's a good choice in terms of accessibility.

I don't like to give a negative answer without offering an alternative solution, which is why I didn't respond earlier. But, no, using tooltips to display form errors typically isn't a good practice from an accessibility standpoint.
